src/js/AppHeader.js

```diff
@@ -35,7 +60,11 @@ class AppHeader extends React.Component {
         const themeClass = this.props.theme ? 'dark-theme' : 'light-theme';
         var modalClass = themeClass + " alertOverlay"
         var isShowingMenu = this.props.router.isActive('/inappmenu')
-        const icon = this.props.appIcon == 'false' ? (<div />) : <MenuIcon isShowingMenu={isShowingMenu}/> ;
+
+        const icon = this.props.appIcon == 'false' ? (<div />) : 
```

Comment:
Can appIcon be renamed to just `icon`

Reply:
yes, done in https://github.com/smartdevicelink/generic_hmi/pull/240/commits/da5e2ba127e3604518601fd85ca7e8e7b210e3b0